draft-ietf-rtcweb-jsep.xml

```diff
@@ -1417,10 +1417,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
             <t>A rollback is performed by supplying a session
             description of type "rollback" with empty contents to
-            either setLocalDescription or setRemoteDescription,
-            depending on which was most recently used (i.e. if the new
-            offer was supplied to setLocalDescription, the rollback
-            should be done using setLocalDescription as well).</t>
+            either setLocalDescription or setRemoteDescription.
+            The effects MUST be the same regardless of whether
+            setLocalDescription or setRemoteDescription is called. Note
+            that the only reason rollbacks use setLocalDescription
+            and setRemoteDescription rather than an independent method
+            is for compatibility with existing implementations.</t>
```

Comment:
Nevermind, I didn't notice this was already revised and committed in https://github.com/rtcweb-wg/jsep/commit/6f5dc414265cab8b76b25e5cd381a53bcb9cd5c5.